packages/flutter/test/widgets/bottom_navigation_bar_test.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
The test for this should be in material, not widgets. There is already a test file there.